### PR TITLE
[Java][MSF4J] add CI test for MSF4J Petstore server

### PR DESCRIPTION
**Pull request by wing328**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [x] Ran the shell/batch script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates)
- [x] Filed the PR against the correct branch: master for non-breaking changes and `2.3.0` branch for breaking (non-backward compatible) changes.

### Description of the PR

For https://github.com/swagger-api/swagger-codegen/issues/5219. Also fixed indentation in pom.xml

### CODE REVIEW

**Comment by ePaul** [comment]
Could you try to update the MSF4J samples again? When I try it, it seems like your changes here are overwritten again (which defeats the purpose of the test).

File: samples/server/petstore/java-msf4j/pom.xml [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
-    </parent> 
+  <parent>
+    <groupId>org.wso2.msf4j</groupId>
+    <artifactId>msf4j-service</artifactId>
+    <version>2.0.0</version>
+  </parent> 
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.swagger</groupId>
-  <artifactId>swagger-msf4j-server</artifactId>
+  <artifactId>swagger-jaxrs-server</artifactId>
```
Comment (ePaul) [inline samples/server/petstore/java-msf4j/pom.xml]: This looks wrong?

Comment (wing328) [inline samples/server/petstore/java-msf4j/pom.xml]: Right. Let me fix it.

Comment (wing328) [inline samples/server/petstore/java-msf4j/pom.xml]: Fixed via https://github.com/swagger-api/swagger-codegen/commit/05c671aa5a4623b312634f7148ef8ebf41763979

**Pull request merged**